lms/djangoapps/verify_student/tests/test_views.py

```diff
@@ -871,6 +873,14 @@ def _post(self, data):
         url = reverse('verify_student_create_order')
```

Comment:
we previously discussed leaving verify_student's /order/ endpoint named as it is, but we can revisit that decision that if this doesn't feel right.